channels/drdynvc/tsmf/ffmpeg/tsmf_ffmpeg.c

```diff
@@ -89,6 +99,7 @@ static tbool tsmf_ffmpeg_init_audio_stream(ITSMFDecoder* decoder, const TS_AM_ME
 	mdecoder->codec_context->channels = media_type->Channels;
 	mdecoder->codec_context->block_align = media_type->BlockAlign;
 
+#if LIBAVCODEC_VERSION_MAJOR < 55
```

Comment:
same applies here.
lets make a wrapper for `av_set_cpu_flags_mask` and backport its code for older api.
(maybe even copy their implementation)